routes/api.php

```diff
@@ -15,3 +15,12 @@
 
 
 Route::post('/alive', 'App\Http\Controllers\AliveController@showAlive')->name('post.show-alive');
+Route::get('/get-account-balance', 'App\Http\Controllers\ReloadlyController@getBalance')->name('post.get-account-balance'); // The name after the @ sign is the name of the function to call in the controller
+Route::get('/get-transaction-status/{id}', 'App\Http\Controllers\ReloadlyController@getTransactionStatus')->name('post.get-transaction-status'); // The {id} means a variable that will be passed in the URL endpoint, so Laravel identifies that as a parameter passed to it
+Route::get('/get-transaction-history', 'App\Http\Controllers\ReloadlyController@getTransactionHistory')->name('post.get-transaction-history');
+Route::get('/get-access-token', 'App\Http\Controllers\ReloadlyController@getAccessToken')->name('post.get-access-token');
```

Comment:
Most likely, you would not want to expose this in your final stage, because the user of the API does not want to deal with Reloadly token. He or she already has a zeepay token to access your api usually. The idea is to hide all the things they wouldn't need to deal with and only expose those that they need. But for testing the Reloadly Service for getting the token, you can use this.